src/zjs_pwm.json

```diff
@@ -7,7 +7,5 @@
     },
     "zjs_config": ["-DBUILD_MODULE_PWM"],
```

Comment:
We need to add "-DENABLE_INIT_FINI" to this list for each module, or to a central place where such flags are specified, otherwise `JERRYX_NATIVE_MODULE ()` will not expand to a constructor/destructor pair, but rather to a pair of global functions.

Reply:
zjs_common.json is used by all of them, I added it there.